src/omnisharp/server.ts

```diff
@@ -319,7 +319,8 @@ export abstract class OmnisharpServer {
             cleanupPromise = new Promise<void>((resolve, reject) => {
                 const killer = exec(`taskkill /F /T /PID ${this._serverProcess.pid}`, (err, stdout, stderr) => {
                     if (err) {
-                        return reject(err);
+                        reject(err);
+                        return;
```

Comment:
FWIW, I personally dislike this being on two lines when it could easily be on one. So, unless there's a real semantic difference I'm unaware of, I'd prefer not making these changes.